File: sdkwork-rtc-sdk-kotlin/providers/rtc-sdk-provider-janus/src/main/kotlin/com/sdkwork/rtc/provider/janus/RtcProviderJanusPackageContract.kt
package com.sdkwork.rtc.provider.janus

/**
 * Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
 */
object RtcProviderJanusPackageContract {
    const val PROVIDER_KEY: String = "janus"
    const val PLUGIN_ID: String = "rtc-janus"
    const val DRIVER_ID: String = "sdkwork-rtc-driver-janus"
    const val PACKAGE_IDENTITY: String = "com.sdkwork:rtc-sdk-provider-janus"
    const val STATUS: String = "future-runtime-bridge-only"
    const val RUNTIME_BRIDGE_STATUS: String = "reserved"
    const val ROOT_PUBLIC: Boolean = false
}

File: sdkwork-rtc-sdk-flutter/providers/rtc_sdk_provider_janus/lib/src/rtc_provider_janus_package_contract.dart
/// Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
final class RtcProviderJanusPackageContract {
  static const String providerKey = "janus";
  static const String pluginId = "rtc-janus";
  static const String driverId = "sdkwork-rtc-driver-janus";
  static const String packageIdentity = "rtc_sdk_provider_janus";
  static const String status = "future-runtime-bridge-only";
  static const String runtimeBridgeStatus = "reserved";
  static const bool rootPublic = false;

  const RtcProviderJanusPackageContract._();
}
